release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -2232,6 +2232,25 @@ sourceStrategy:
 
 * Previously, although the reference `PolicyGenTemplate` for `group-du-sno-ranGen.yaml` includes two `StorageClass`  entries, the generated policy included only one. With this update, the generated policy now includes both policies. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2049306[*BZ#2049306*]).
 
+* Previously, attempting to install an {sno} cluster with two Day 2 workers failed when using `ignitionEndpoint`.
+After the initial spoke cluster installation, the `AgentClusterInstall` never reached the `adding-hosts` state and remained in the `installed` state. Additionally, the admin `kubeconfig` secret was not generated on the hub cluster, so the `kubeapi` of the spoke cluster could not be accessed.
+With this update, installing an {sno} cluster with two Day 2 workers succeeds when using `ignitionEndpoint`.
+The `AgentClusterInstall` now reaches the `adding-hosts` state and the admin `kubeconfig` secret is generated on the hub cluster, allowing access to the `kubeapi` of the spoke cluster.
+
+* Previously, when deploying a spoke cluster via Assisted Service (Infra Operator) in a disconnected environment and referencing a public {product-title} image to install, the assisted service pod would not recognize the `ImageContentSourcePo` (ICSP) object and fail to pull the image.
```

Comment:
This one has the same spacing issues as the one above.

Reply:
Spacing is intentional [ventilated prose](https://writetheasciidocs.netlify.app/ventilated-prose). Asciidoc renders around blank lines, so it displays ventilated prose the same as traditionally-wrapped lines.